src/components/pagination.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { Fragment } from 'react';
+
+interface PaginationProps {
+  currentPage: number;
+  neighbours: number;
+  limit: number;
+  setCurrentPage: (page: number) => void;
+  elementsCount?: number;
+  loading?: boolean;
+}
+
+export const Pagination = ({
+  currentPage,
+  neighbours,
+  limit,
+  setCurrentPage,
+  elementsCount,
+  loading,
+}: PaginationProps): React.ReactElement => {
+  if (loading) {
+    return <button>loading pagination...</button>;
```

Comment:
Terá algum tipo de interação nesse loading? Se não tiver, poderia ser substituído por outro?

Reply:
Quando for estilizar os componentes, pretendo criar um componente de loading para os botões

Reply:
Vou substituir por uma tag de texto